deepmd/utils/data.py

```
@@ -501,6 +502,7 @@ def reformat_data_torch(self, data):
             data["box"] = None
         return data
 
+    @lru_cache(1)
```

Ruff (0.8.2)

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)